CSSAnalyzer.py

```diff
@@ -174,8 +174,10 @@ def parseImageSelector(selector, htmlElements, searchInfo, expectedIndex, tag):
    numberElementsFoundWithValue = 0
    expectedValue = searchInfo["value"]
 
-   for selector in htmlElements:
-      if(selector['src'] == expectedValue ):
+   for element in htmlElements:
+      # The element source sometimes comes with a relative path, so it starts with '../'.
+      # This is to remove the dots and compare against the end of expected value. 
+      if(element['src'] == expectedValue or expectedValue.endswith(element['src'][2:None])):
```

Comment:
HI @napoles-mk , what test was failing because of this? This is not related at all to the JS snippet issues, right?

Reply:
Hey @jesus-serranop, this is an edge case I found. The analyzer was not working correctly when the element is an image. 
The use case forr this is a step where user clicks on an image to do a check.

Reply:
Correct, just a small comment that we just released a fix for this today. All the executors were updated. You will have to update and then put this change again.

Reply:
Nice, thanks @jesus-serranop. I have merge. Besides the fix there was another part to check the relative paths from the src. That is still showed in my changes.